src/maintagent/agent/usage.c

```diff
@@ -30,8 +30,8 @@
 FUNCTION void usage(char *name)
 {
   printf("Usage: %s [options]\n", name);
-  printf("  -a   :: Run all non slow maintenance operations.\n");
-  printf("  -A   :: Run all maintenance operations.\n");
+  printf("  -a # :: Run all non slow maintenance operations where # is the retention period of expired tokens.\n");
+  printf("  -A # :: Run all maintenance operations where # is the retention period of expired tokens.\n");
```

Comment:
In my opinion, we can keep the removal of token as a separate process only and not mix with "all jobs". It will also keep the options `-a` and `-A` backwards compatible.

What do you think @ag4ums ?